kotlinx-coroutines-core/jvm/test/knit/ClosedAfterGuideTestExecutor.kt

```diff
@@ -44,7 +44,10 @@ private class ClosedAfterGuideTestDispatcher(
     }
 
     override fun close() {
-        (executor as ExecutorService).shutdown()
+        (executor as ExecutorService).apply {
+            shutdown()
+            awaitTermination(1, TimeUnit.MINUTES)
+        }
```

Comment:
No strong opinion about this change, I just found it severely surprising that there's no way to wait for the tasks to finish executing. Can undo this, move the corresponding test to native-only, and change the doc to state that the previously-submitted tasks may execute *in parallel* to the caller of `close()`.